src/popup/container.js

```diff
@@ -110,7 +111,7 @@ class PopupContainer extends Component {
         const dbResult = await lookup(pageId)
         const result = {
             loggable: isLoggable({ url }),
-            bookmark: dbResult.bookmarks.size !== 0,
+            bookmark: dbResult === null ? false : dbResult.bookmarks.size !== 0,
```

Comment:
best to use a non-strict equality check when checking missing things using `null`. `=== null` will only match `null` while `== null` matches `undefined`, `null` (in this case it should actually be `null` but that is implementation-dependent on `initSingleLookup`)